superchain/internal/codegen/main.go

```diff
@@ -70,12 +70,13 @@ func main() {
 	fmt.Printf("Found %d chains...\n", len(allChains))
 
 	currentDir := currentDir()
+	repositoryRoot := filepath.Join(currentDir, "../../../")
```

Comment:
There's a more portable method to find the root, which works from any cwd, see [here](https://github.com/ethereum-optimism/optimism/blob/457f33f4fdda9373dcf2839619ebf67182ee5057/op-service/util.go#L126). But this works for now too.

Reply:
Hmm looks like a useful tool -- but I don't think it would work out of the box in this repo (we have a `go.work` file at the top level, and several `go.mod` files one level down). 

We can revisit if it ends up being a pain to maintain this code, I think it will be rare that we will move things around.